src/views/Dashboard/styles.scss

```diff
@@ -0,0 +1,10 @@
+div.dashboard__flexbox{
+    display: flex;
+    flex-wrap: nowrap;
+    width:100%;
+}
+
+div.bill__display{
+    width: calc(100% - 302px);
+    margin-left: 302px;
```

Comment:
Can you pull from the release branch to get the sidebar? Then see if this value still holds

Reply:
yes this matches I looked at it before